index.json

```diff
@@ -4424,6 +5677,13 @@
       "nightlyUrl": "https://w3c.github.io/accname/"
     },
     "seriesVersion": "1.2",
+    "organization": "W3C",
+    "groups": [
+      {
+        "name": "Devices and Sensors Working Group",
```

Comment:
This is wrong, so there must be a bug somewhere :)

(222 specs are assigned to DAS)

Reply:
Such a productive working group...

Reply:
for reference, here is the outlook with the fixed code
```
    6 "Accessible Rich Internet Applications Working Group"
      2 "Audio Working Group"
      2 "Browser Testing and Tools Working Group"
    113 "Cascading Style Sheets (CSS) Working Group"
     15 "Devices and Sensors Working Group"
      1 "Distributed Tracing Working Group"
      1 "Geolocation Working Group"
      2 "GPU for the Web Working Group"
      2 "HTML Media Extensions Working Group"
      2 "Immersive Web Community Group"
      8 "Immersive Web Working Group"
      1 "Internationalization Working Group"
      1 "Math Working Group"
      5 "Media Working Group"
      1 "Pointer Events Working Group"
      2 "Privacy Community Group"
      3 "Second Screen Working Group"
      1 "Service Workers Working Group"
      9 "SVG Working Group"
     17 "TC39"
      2 "Technical Architecture Group"
      1 "Timed Text Working Group"
      1 "Tracking Protection Working Group"
     14 "Web Application Security Working Group"
     19 "Web Applications Working Group"
      3 "WebAssembly Working Group"
      1 "Web Authentication Working Group"
      1 "Web Bluetooth Community Group"
      1 "Web Cryptography Working Group"
      1 "Web Events Working Group"
      1 "Web Fonts Working Group"
     44 "WebGL"
      1 "Web Machine Learning Working Group"
      1 "Web NFC Community Group"
      5 "Web Payments Working Group"
     18 "Web Performance Working Group"
     64 "Web Platform Incubator Community Group"
      4 "Web Platform Working Group"
     16 "Web Real-Time Communications Working Group"
      1 "WebTransport Working Group"
     14 "WHATWG"
```
based on `jq '.[].groups[].name' index.json|sort|uniq -c`